routes/resources.js

```diff
@@ -5,22 +5,129 @@
  * See: https://expressjs.com/en/guide/using-middleware.html#middleware.router
  */
 
-const express = require('express');
-const router  = express.Router();
+const express = require("express");
+const router = express.Router();
 
 module.exports = (database) => {
-  router.get('/', (req, res) => {
+  // 1. get all comments of a resource
+  router.get("/:id/comments", (req, res) => {
+    const { id } = req.params;
+
+    database
+      .getResourceComments(id)
+      .then((resources) => res.send({ resources }))
+      .catch((e) => {
+        console.error(e);
+        res.send(e);
+      });
+  });
+
+  // 2. get average rating of a resource
+  router.get("/:id/rating", (req, res) => {
+    const { id } = req.params;
+
+    database
+      .getResourceRating(id)
+      .then((resources) => res.send({ resources }))
+      .catch((e) => {
+        console.error(e);
+        res.send(e);
+      });
+  });
+
+  // 3. get usernames of users who liked a resource
+  router.get("/:id/likes", (req, res) => {
```

Comment:
I'm not sure this is the correct nomenclature. This is like "likes by a specific user specified by :id"